web/src/user_group_pill.ts

```diff
@@ -47,32 +59,36 @@ export function get_user_ids(pill_widget) {
     return user_ids;
 }
 
-export function append_user_group(group, pill_widget) {
+export function append_user_group(group: UserGroup, pill_widget: UserGroupPillWidget): void {
     if (group !== undefined && group !== null) {
         pill_widget.appendValidatedData({
             type: "user_group",
             display_value: display_pill(group),
             id: group.id,
+            group_name: group.name,
```

Comment:
Could you add a test to verify this change? Looks like we previously didn't have coverage for `append_user_group` and missed this bug.

Reply:
Yeah, this was definitely a bug before this change, added test coverage for this function in a new commit.